beacon-chain/sync/rpc.go

```diff
@@ -76,3 +77,16 @@ func (r *RegularSync) registerRPC(topic string, base proto.Message, handle rpcHa
 		}
 	})
 }
+
+func copyValues(msg interface{}) interface{} {
+	switch t := msg.(type) {
+	case proto.Message:
+		return proto.Clone(t)
+	case uint64:
```

Comment:
Do you need these cases? Wouldn’t it work with the default/default option?

Reply:
this was a workaround to get the tests to pass, once #3512 is merged in I will just remove these cases and use that instead